tests/integration/lag/11-mlag-anycast.yml

```diff
@@ -0,0 +1,54 @@
+message: |
+  The devices under test are an MLAG pair sharing an anycast gateway and
+  connected to an Arista EOS switch.
+
+  The Arista EOS switch should have a working LAG and the attached host should
+  be able to ping the anycast gateway.
+
+groups:
+  _auto_create: true
+  switches:
+    members: [ s1, s2 ]
+    module: [ lag, vlan, gateway ]
+  probes:
+    members: [ x1 ]
+    device: eos
+    module: [ lag, vlan ]
+  hosts:
+    members: [ h1 ]
+    device: linux
+    provider: clab
+
+vlans:
+  red:
+    gateway.protocol: anycast
+    gateway.id: 1
+    links: [ x1-h1 ]
+    prefix.ipv4: 172.16.42.0/24
+
+links:
+- lag:
+    members: [s1-s2]
+    mlag.peergroup: true
+- lag.members: [ x1-s1, x1-s2 ]
+  vlan.access: red
+
+validate:
+  lag:
+    description: Check LAG ports on X1
+    wait: 30
+    wait_msg: Waiting for LAG to start
+    nodes: [ x1 ]
+    devices: [ eos ]
+    pass: The LAG is active
+    fail: X1 cannot establish a LAG with both switches in MLAG pair
+    show: port-channel | json
+    valid: >-
+      len(portChannels['Port-Channel1'].activePorts) == 2
+
+  ping:
+    description: End-to-end connectivity test
+    nodes: [ h1 ]
+    wait_msg: Waiting for STP to enable the ports
```

Comment:
Dell OS10 has a (default) 90 second delay-restore timer for MLAG, we may be waiting for that here too

Reply:
Can you reduce that delay (who has time to wait forever for vendor ****?) Otherwise I can extend the wait period for OS10.